internal/iam/command.go

```diff
@@ -32,8 +33,22 @@ func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
 	cmd.AddCommand(newPoolCommand(prerunner))
 	cmd.AddCommand(newProviderCommand(prerunner))
 	cmd.AddCommand(newRBACCommand(cfg, prerunner))
+	cmd.AddCommand(newIpFilterCommand(prerunner))
+	cmd.AddCommand(newIpGroupCommand(prerunner))
 	cmd.AddCommand(newServiceAccountCommand(prerunner))
 	cmd.AddCommand(newUserCommand(prerunner))
 
 	return cmd
 }
+
+func AddAndDeleteResourceWarning(resource string, l *log.Logger) {
```

Comment:
The current naming sounds like the warning itself is being added/deleted.
```suggestion
func WarnAddAndDeleteResource(resource string, l *log.Logger) {
```